README.md

```diff
@@ -105,6 +105,7 @@ Supported Components
     -   [cephfs-provisioner](https://github.com/kubernetes-incubator/external-storage) v1.1.0-k8s1.10
     -   [cert-manager](https://github.com/jetstack/cert-manager) v0.3.2
     -   [ingress-nginx](https://github.com/kubernetes/ingress-nginx) v0.16.2
+    -   [coredns](https://github.com/coredns/coredns) v1.1.2
```

Comment:
the right version is v1.1.3 : https://github.com/kubernetes/kubernetes/pull/64258

Reply:
thanks